README.md

```diff
@@ -58,26 +52,25 @@ Argument Reference
 
 The following arguments are supported:
 
-* `domain` - (Required) Domain name of the certificate
+* `domain` - (Required) Domain name of the certificate.
 
-* `hostname` - (Required) Hostname of the certificate
+* `hostname` - (Required) Hostname of the certificate.
 
 * `validation_method` - Which method to use for validation. DNS or EMAIL are valid, NONE can be used for certificates that were imported into ACM and then into Terraform.
 
 * `subject_alternative_names` - A list of domains that should be SANs in the issued certificate
 
-* `skip_route53_validation` - Set to true for zones not hosted in Route53
+* `skip_route53_validation` - Skip Route53 validation. Default is false, which creates Route53 domain validation records to be created. Set to true for zones not hosted in Route53.
```

Comment:
The convention we generally followed was to put the default at the end. I suggest something like this:

* `skip_route53_validation` - Set to true to disable Route53 domain validation record creation. This is useful for zones not hosted in Route53 (Default is false).

Of if you want to keep it short:

* `skip_route53_validation` - Set to true for zones not hosted in Route53. (Default is false).